src/components/OptionsSelector.js

```diff
@@ -231,7 +221,7 @@ class OptionsSelector extends Component {
             focusedIndex: newFocusedIndex,
         });
 
-        if (newOptions.length <= newFocusedIndex) {
+        if (newOptions.length <= newFocusedIndex || !_.isEqual(this.props.selectedOptions, prevProps.selectedOptions)) {
```

Comment:
Sorry, I don't quite follow. Mind explaining why you added this?

Reply:
Good question ... I don't remember. Removed it and everything seems to work as expected. Maybe it's not needed since we merged https://github.com/Expensify/App/pull/9714 ?